src/Definition/Common.php

```diff
@@ -58,6 +72,22 @@ public function isNullable()
     /**
      * @return string
      */
+    public function getDefault()
```

Comment:
Doesn't this https://github.com/keboola/connection/pull/788 use `getSQLDefinition`?